lib/benchee/formatters/console.ex

```diff
@@ -147,10 +148,11 @@ defmodule Benchee.Formatters.Console do
   end
 
   defp column_descriptors(label_width) do
-    "\n~*s~*s~*s~*s~*s\n"
+    "\n~*s~*s~*s~*s~*s~*s\n"
     |> :io_lib.format([-label_width, "Name", @ips_width, "ips",
                        @average_width, "average",
-                       @deviation_width, "deviation", @median_width, "median"])
+                       @deviation_width, "deviation", @median_width, "median",
+                       @percentile_width, "P99"])
```

Comment:
him `P99` seems weird to me, `99th` or so might be better or wdyt?

Reply:
I went back and forth on this: `99th percentile` is too long. I chose `P99` because it's actually a thing people use (according to the internet). But they are probably professional statisticians. I like 99th — we just have to be sure to handle 1st, 2nd, and 3rd too

Other options:

- 99th
- 99%
- 99th %
- 99th %ile

I'll extract it to a function so we can change it easily

Reply:
`99%` looks even better to me :+1:

Reply:
Maybe `99th %`? I'll keep thinking of other possibilities, too, but that's the only one that jumped out to me.